### DISCUSSION TITLE
Shows error when creating new environment

### DISCUSSION THREAD

**Discussion by puskarrr** [giswqs/geog-312#147] [Q&A] [answered]
![Screenshot 2024-11-10 222656](https://github.com/user-attachments/assets/171e4c73-b3ef-4d14-897d-77ae2d5a0ac1)
![Screenshot 2024-11-10 222707](https://github.com/user-attachments/assets/2e8d4fef-4c02-4f0b-a196-da7ccb494a55)
It shows cannot find suitable path for TLS CA certificate bundle. The invalid path shown in that picture is no longer in my pc.

**Comment by giswqs** [comment, marked as answer]
This is Anaconda issue. You can Google it to find solutions. I don't use Windows. Not much I can help.
